### PR TITLE
[MIG] stock-scanner: Migration to 12.0

### PR DESCRIPTION
**Pull request by fmdl**
[FIX] duplicate scenario

### CODE REVIEW

**Comment by fmdl** [comment]
@sla-subteno-it

**Review by rousseldenis — APPROVED** [review]
Code review. Seems good. @fmdl Thanks for improvements!

**Comment by tbaden** [comment]
thanks for the work 👍 
I did a quick check and the xml files should be refactored (see contribution guidelines)
https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst

**Comment by fmdl** [comment]
@tbaden was is the exact issue with xml files ?

**Comment by tbaden** [comment]
@fmdl 
id pattern

> Use the following patterns, where <model_name> is the name of the model that the menu, view, etc. belongs to (e.g. for a res.users form view, the name would be res_users_view_form):
> 
> For a menu: <model_name>_menu
> For a view: <model_name>_view_<view_type>, where view_type is kanban, form, tree, search, ...
> For an action: the main action respects <model_name>_action. Others are suffixed with _<detail>, where detail is an underscore lowercase string explaining the action (should not be long). This is used only if multiple actions are declared for the model.
> For a group: <model_name>_group_<group_name> where group_name is the name of the group, generally 'user', 'manager', ...
> For a rule: <model_name>_rule_<concerned_group> where concerned_group is the short name of the concerned group ('user' for the 'model_name_group_user', 'public' for public user, 'company' for multi-company rules, ...).


 and id should be the first attribute
> When declaring a record in XML:
> 
> Place id attribute before model

**Comment by fmdl** [comment]
@pedrobaeza I need to update the xml file ?

**Comment by pedrobaeza** [comment]
@fmdl @tbaden well, it's something desirable for being homogeneous, but not mandatory for blocking the PR, so if you want to do it, it would be welcome. What I see more advisable is to squash all your migration commits in 1, as they don't add any value to be split.

**Comment by tbaden** [comment]
@pedrobaeza 
how should I handle these kind of changes in the future?
just suggesting these changes and approve anyway?
and which kind of changes described in the contribution guideline are really "pr block worthy"?

@fmdl will approve when squashed

**Review by tbaden — APPROVED** [review]
lgtm now

File: stock_scanner/__manifest__.py
Comment (tbaden) [inline stock_scanner/__manifest__.py]: website should be: https://github.com/OCA/stock-logistics-barcode

**Review by tbaden — CHANGES_REQUESTED** [review]
noticed this just now

File: stock_scanner/__manifest__.py
Comment (fmdl) [reply stock_scanner/__manifest__.py]: done

**Review by tbaden — APPROVED** [review]

**Pull request merged**
